### ISSUE TITLE
Models: Don't wrap anyOf, oneOf, allOf in curly brackets 

### ISSUE THREAD

**Issue by crussell52** [swagger-api/swagger-ui#4088]
| Q                               | A
| ------------------------------- | -------
| Bug or feature request?         |  Feature (Enhancement)
| Which Swagger/OpenAPI version?  | 3.0.x
| Which Swagger-UI version?       | Docker: swaggerapi/swagger-editor@sha256:3b583ca9e0f22caae4d8cd16752dc3f98f86b13d722b6d7cda6e84adf204a35e
| How did you install Swagger-UI? | Docker
| Which browser & version?        | Chrome Version 63.0.3239.132 (Official Build) (64-bit)
| Which operating system?         | MacOS

### Demonstration API definition

```yaml
openapi: 3.0.0
info:
  title: Test API
  version: 0.0.1
servers:
  - url: 'localhost:3000/api'
    description: Local API URL.
paths:
  /test:
    get:
      requestBody:
        content:
          application/json:
            schema: {}

      responses:
        '200':
          description: OK
components:
  schemas:
    Sample:
      oneOf:
        - type: integer
        - type: boolean

    SampleObject:
      type: object
      properties:
        foo:
          type: integer
```

### Expected Behavior

When rendering Models don't wrap anyOf, allOf, or oneOf in `{}`

### Current Behavior

![image](https://user-images.githubusercontent.com/375294/34729975-1420db8a-f52c-11e7-852d-9fa21c8437b2.png)
![image](https://user-images.githubusercontent.com/375294/34729993-240b7262-f52c-11e7-9b99-14830a78318f.png)

### Context

Wrapping `oneOf` (etc) in `{ }` is in conflict with using `{ }` to represent an Object.  It implies one of the models is embedded within an Object.

![image](https://user-images.githubusercontent.com/375294/34730279-eb35c1ee-f52c-11e7-8a8a-246ad4908213.png)

This is especially evident when the `oneOf` element is collapsed:
![image](https://user-images.githubusercontent.com/375294/34730291-f5f92c42-f52c-11e7-9c23-9315a8f52ff0.png)

**Comment by expresspotato** [comment]
Yes, left my devs puzzled how they can create json with just the field value...
